### PR TITLE
Add more animation options to Ambient Animation module

### LINKED ISSUE #311: Ambient animations included with Achilles 1.3.0 not in ZEN
Achilles update 1.3.0 introduced some extra ambient animations that were very useful for mission creation, such as the "dead" animation for wounded characters leaning against walls. This is not present in ZEN and it would be nice to have more of these animations.

### PR DESCRIPTION
**Pull request by SDTheCyanWyan**
**When merged this pull request will:**
 - Close #311 

Adds more animations such as:
- Sitting on Floor with Weapons,
- Standing Idle with Pistol,
- Squatting Unarmed and Armed,
- Guard while Kneeling,
- Wounded Leaning,
- Surrender,
- Dead Lying down, Leaning, Sitting,
- Leaning on Table,
- Looking through Binoculars

### CODE REVIEW

File: addons/modules/stringtable.xml [hunk 1]
```diff
@@ -218,6 +267,41 @@
             <Polish>Naprawianie pojazdu (Na leżąco)</Polish>
             <Japanese>車両の修理 (寝転がり)</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAmbientAnim_Dead">
+            <English>Dead</English>
+            <Russian> </Russian>
```
Comment (bux) [inline addons/modules/stringtable.xml]: remove empty entries

Comment (SDTheCyanWyan) [reply addons/modules/stringtable.xml]: Fixed them, thanks

Comment (bux) [reply addons/modules/stringtable.xml]: Thanks

File: addons/modules/stringtable.xml [hunk 2]
```diff
@@ -57,13 +66,23 @@
             <Polish>Bezczynne stanie (bez broni)</Polish>
             <Japanese>立ち姿勢のアイドル (武器なし)</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAmbientAnim_StandWithPistol">
+            <English>Stand Idle (Pistol)</English>
+            <Russian> </Russian>
+            <German> </German>
+            <Polish> </Polish>
+            <Japanese> </Japanese>
```
Comment (mharis001) [inline addons/modules/stringtable.xml]: Remove these.

Comment (SDTheCyanWyan) [reply addons/modules/stringtable.xml]: Done, missed those, thanks

**Comment by SDTheCyanWyan** [comment]
Added extra animations to Sit On Floor and Lean

File: addons/modules/stringtable.xml [hunk 3]
```diff
@@ -218,6 +239,21 @@
             <Polish>Naprawianie pojazdu (Na leżąco)</Polish>
             <Japanese>車両の修理 (寝転がり)</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAmbientAnim_Dead">
+            <English>Dead</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAmbientAnim_DeadLeaned">
+            <English>Dead (Leaned)</English>
```
Comment (CreepPork) [inline addons/modules/stringtable.xml]: Leaned doesn't sound right, at least to me.
```suggestion
            <English>Dead (Leaning)</English>
```

Comment (SDTheCyanWyan) [reply addons/modules/stringtable.xml]: Changed Wounded (Leaned) and Dead (Leaned) to Leaning. Thanks.

**Comment by CreepPork** [comment]
You pushed the wrong commits to the PR. Revert the latest commit.

**Comment by SDTheCyanWyan** [comment]
> You pushed the wrong commits to the PR. Revert the latest commit.

Sorry about that, fixed and reverted it.

File: addons/modules/functions/fnc_moduleAmbientAnim.sqf [hunk 4]
```diff
@@ -56,17 +61,25 @@ if (isPlayer _unit) exitWith {
         LSTRING(ModuleAmbientAnim_ShowVehicleTheWay),
         LSTRING(ModuleAmbientAnim_TreatWounded),
         LSTRING(ModuleAmbientAnim_CombatWounded),
+        LSTRING(ModuleAmbientAnim_WoundedLeaned),
```
Comment (mharis001) [inline addons/modules/functions/fnc_moduleAmbientAnim.sqf:64]: Move the leaning,option below genera. Also, change `WoundedLeaned` to `WoundedLeaning`.

File: addons/modules/functions/fnc_moduleAmbientAnim.sqf [hunk 5]
```diff
@@ -56,17 +61,25 @@ if (isPlayer _unit) exitWith {
...
         LSTRING(ModuleAmbientAnim_ShockedCivilian),
         LSTRING(ModuleAmbientAnim_HidingCivilian),
+        LSTRING(ModuleAmbientAnim_Surrender),
         LSTRING(ModuleAmbientAnim_SitCaptured),
         LSTRING(ModuleAmbientAnim_RepairStand),
         LSTRING(ModuleAmbientAnim_RepairKneel),
-        LSTRING(ModuleAmbientAnim_RepairProne)
+        LSTRING(ModuleAmbientAnim_RepairProne),
+        LSTRING(ModuleAmbientAnim_Dead),
+        LSTRING(ModuleAmbientAnim_DeadLeaned),
```
Comment (mharis001) [inline addons/modules/functions/fnc_moduleAmbientAnim.sqf:78]: `DeadLeaning`.

**Comment by SDTheCyanWyan** [comment]
> New animations work as expected, some minor cleanup suggestions.

What kind of clean ups would you recommend? Making names of new animations better/more consistent, reorganizing list, etc?

**Pull request merged**